docs/20_smart-contracts/10_getting-started/10_dune-guide.md

```diff
@@ -2,153 +2,150 @@
+[Docker Utilities for Node Execution (DUNE)](https://github.com/AntelopeIO/DUNE) is a client tool. 
+As a blockchain developer or node operator, use DUNE for your common tasks.
 
-Before getting started with smart contract development, you need to learn about DUNE and how to install it on your platform.
+Before you begin, learn about DUNE and how to install it on your platform.
 
 ### Installation
 
-DUNE can be installed and run on the following platforms:
+You can install and run DUNE n the following platforms:
 * Linux
 * Windows
 * MacOS
 
-Installation instructions for each supported platform are available on the [DUNE's github project](https://github.com/AntelopeIO/DUNE) page.
+Each platform has their own installation instructions. Click [DUNE's GitHub project](https://github.com/AntelopeIO/DUNE) to view the instructions.
 
-Once you're done, you can run `dune --help` to see a list of all supported commands.
+Run `dune --help` to view a list of all supported commands.
 
 ## Wallets
 
-DUNE handles wallet management for you so that you don't have to. 
+DUNE handles wallet management for you so you don't have to. 
 
 If you need to import a new key into your wallet:
 
 ```shell
 dune --import-dev-key <PRIVATE_KEY>
 ```
 
-## Node Management
+## Node management
 
-Creating a new local EOS blockchain is easy with DUNE.
+DUNE easily creates a new local EOS blockchain.
+
+The following command creates a new node called `NODE_NAME`.  This node initiates with default settings. These default settings configure the node to serve as an API/producer node. You can deploy smart contracts to it and perform tests on it.
 
 ```shell
 dune --start <NODE_NAME>
 ```
 
-The command above creates a new node called `NODE_NAME` and starts it with default settings. 
-The node is configured to serve as an API/producer node that you can deploy smart contracts to, and perform tests on.
-
-> ❔ **Errors**
+> 📝 **In Case of Errors**
 >
 > You may see errors at the end of the node setup process.
-> If you do you can refer to this guide to troubleshoot common errors, or reach out to us on our
+> If you do, refer to this guide to troubleshoot common errors, or reach out to us on our
 > [Telegram channel](https://t.me/antelopedevs) for help.
 
-You can see a list of EOS nodes on your system:
+The following command shows you a list of EOS nodes on your system:
 
 ```shell
 dune --list
 ```
 
-You can also check if your active node's RPC API is live:
+The following command checks if your active node's RPC API is live:
 
 ```shell
 dune -- cleos get info
 ```
 
-To shut down your node:
+The following command shuts down your node:
 
 ```shell
 dune --stop <NODE_NAME>
 ```
 
-To remove a node entirely:
+The following command removes a node:
```

Comment:
It was already understood that the command is what causes the action. 

We don't need to write `The following command` 4 times. It is redundant and wastes the developer's time.

Reply:
Done